composer.json

```diff
@@ -29,7 +29,7 @@
         "source": "https://github.com/browscap/browscap-php"
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
Please could you add 8.4 to the pipeline too so the tests are verified working too?